sdk/identity/identity/src/client/identityClient.ts

```diff
@@ -113,8 +117,11 @@ export class IdentityClient extends ServiceClient {
     });
 
     try {
-      return await this.sendTokenRequest(webResource, expiresOnParser);
+      const tokenResponse = await this.sendTokenRequest(webResource, expiresOnParser);
```

Comment:
Why aren't we setting the spanOptions on the `webResource` that is created here?

Reply:
I missed it. Added it now.